app/controllers/chart_instances_controller.rb

```diff
@@ -4,6 +4,35 @@ class ChartInstancesController < ApplicationController
   include ERB::Util
   include Scalarm::ServiceCore::ParameterValidation
 
+
+=begin
+apiDoc:
+  @api {get} /chart_instances/:id Chart rendering
+  @apiName chart_instances#show
+  @apiGroup ChartInstances
+  @apiDescription When called, firstly validates chart name (:id) and then escape all other parameters for safety. Next, invoke visualisation class method handler (located at /app/visualisation_methods/plugin in application) to prepare data for loaded draw function content.
```

Comment:
@Andriu01 - można by zrobić tak, że w pierwszej linijce piszemy to co najważniejsze, czyli co otrzymujemy dzięki tej metodzie - Returns HTML header (dla mnie "header" jest utaj niezrozumiałe, bo miesza mi się z ``<head>`` w HTML...) itd., a potem pozostałe zdania, które tutaj napisałeś. Chodzi o to, że użytkownika tej usługi najbardziej interesuje to co dostanie.

Jeśli zrobisz tak, to wyrenderuje się tak jak napisałem:
```
  @apiDescription
    Returns HTML header rendered from file (located at /app/visualisation_methods/<chart_name>/chart in application) which adds new window to modal with generated chart. It also has a hide button with JavaScript function which handle click event.

  When called, firstly validates chart name (:id) and then escape all other parameters for safety. Next, invokes visualisation class method handler (located at /app/visualisation_methods/plugin in application) to prepare data for loaded draw function content.
```

Poza tym, zwróć uwagę na użycie "s" na końcu czasowników, żeby było wiadomo co podejmuje akcje.